src/app/apis/details.js

```diff
@@ -0,0 +1,46 @@
+const LOAD = 'reactcapstone/apis/LOAD';
+
+const init = [];
+
+export function load(country) {
+  return {
+    type: LOAD,
+    country,
+  };
+}
+
+const getApiCountry = async () => {
+  const resp = await fetch('https://disease.sh/v3/covid-19/countries');
+  const data = resp.json();
+
+  return data;
+};
+
+export const displayCountryInfo = () => async (dispatch) => {
+  const countryData = await getApiCountry();
+
+  const countriesArray = countryData.map((ele) => ({
+    // eslint-disable-next-line no-underscore-dangle
```

Comment:
- [ ] please avoid turning off eslint, linters are there to make your code more efficient and readable.